static/app/views/settings/account/notifications/utils.tsx

```diff
@@ -53,7 +53,17 @@ export const getFallBackValue = (notificationType: string): string => {
       return 'committed_only';
     case 'workflow':
       return 'subscribe_only';
+    case 'approval':
+      return 'always';
+    case 'quota':
+      return 'always';
+    case 'spikeProtection':
+      return 'always';
+    case 'reports':
+      return 'always';
     default:
+      // issue, quotaErrors, quotaTransactions, quotaAttachments,
+      // quotaReplays, quotaWarnings, quotaSpendAllocations
```

Comment:
Are these cases that still need to be handled or cases where the default is ''?

Reply:
The latter, I'll clarify